irods_browser_app/static/js/irods.js

```diff
@@ -0,0 +1,236 @@
+$('#btn-signin-irods').on('click',function(){
+    if(sessionStorage.username)
+        $("#username").val(sessionStorage.username);
+    if(sessionStorage.password)
+        $("#password").val(sessionStorage.password);
+    if(sessionStorage.host)
+        $("#host").val(sessionStorage.host);
+    if(sessionStorage.port)
+        $("#port").val(sessionStorage.port);
+    if(sessionStorage.zone)
+        $("#zone").val(sessionStorage.zone);
```

Comment:
you should probably prefix or combine all these as well. If another part of the site tries to use any of these for some other purpose, you'll run into weird bugs mixing data. Without any kind of grouping or prefixing here, someone else adding something to sessionStorage is unlikely to even be aware of this code.